index.html

```diff
@@ -25,6 +26,16 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+      <nav id="mobile-nav">
+        <span>x</span>
+        <ul>
+					<li><a class="nav-link" href="#">Portfolio</a></li>
+					<li><a class="nav-link" href="#">About</a></li>
+					<li>
+						<a class="nav-link" href="#">Contact</a>
+					</li>
+				</ul>
+			</nav>
```

Comment:
The bottom part of the mobile menu should look like this:

<img width="225" alt="image" src="https://user-images.githubusercontent.com/87897753/158901600-4ad4dfc1-af87-4b38-b48f-f4c41dc77622.png">

<img width="217" alt="image" src="https://user-images.githubusercontent.com/87897753/158901768-052629dc-7ac7-447d-8778-b13b47d49b49.png">